src/main/java/robot/subsystems/drivetrain/pure_pursuit/Path.java

```diff
@@ -298,9 +320,19 @@ private double recursiveDistance(int i) {
     }
 
     /**
-     * Attributes to all points their curvature in correlation to their adjacent points.
+     * Attribute to all points their curvature in correlation to their adjacent points.
+     * sets the curvature of each point and saves as a parameter, so that it doesnt have to be calculated real time.
```

Comment:
Use the  imperative form. Also, a few tweaks.
```suggestion
     * set the curvature of each point and save as a parameter, so that it doesn't have to be calculated real time.
```
What does "save as a parameter" mean?